### ISSUE TITLE
quick question about `BertForMaskedLM`

### ISSUE THREAD

**Issue by h56cho** [huggingface/transformers#7643]
Hello,
I have a question about the example code that can be found in the documentation for `BertForMaskedLM` model. The example from the documentation is shown below:
```python
from transformers import BertTokenizer, BertForMaskedLM
import torch
tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
model = BertForMaskedLM.from_pretrained('bert-base-uncased', return_dict=True)
input_ids = tokenizer("Hello, my dog is cute", return_tensors="pt")["input_ids"]
outputs = model(input_ids, labels=input_ids)
loss = outputs.loss
prediction_logits = outputs.logits
```
In this example, from the input string "Hello, my dog is cute", I don't see any `mask_token` in it. Also, the example code simply passes the label as `label=input_ids`. 

So in this particular example, how exactly does the `BertForMaskedLM` model calculate the masked-LM loss (since the `mask_token` is not specified in the input string)? when I simply pass `labels = input_ids`, does `BertForMaskedLM` model automatically place the `mask_token` over the the first token of the input string (or something similar to this)?

I don't think that the code provided in the documentation is wrong, because when I run the code on my machine, it runs smoothly without generating any error.

Thank you,

**Comment by sgugger** [comment]
This has been fixed already but is only visible in the master documentation: see [here](https://huggingface.co/transformers/master/model_doc/bert.html#bertformaskedlm). The documentation that is shown by default corresponds to the last release and the fix in the docstrings has been done since then :-)
